app/controllers/synapses_controller.rb

```diff
@@ -47,9 +49,11 @@ def update
     @synapse = Synapse.find(params[:id])
     @synapse.desc = '' if @synapse.desc.nil?
     authorize @synapse
+    @synapse.updated_by = current_user
+    @synapse.assign_attributes(synapse_params)
```

Comment:
which it should be, because then you could mock as if someone else made the change